vars/openShiftUtils.groovy

```diff
@@ -78,8 +66,8 @@ def withNode(Map parameters = [:], Closure body) {
     */
     def image = parameters.get('image', pipelineVars.iqeCoreImage)
     def cloud = parameters.get('cloud', pipelineVars.defaultCloud)
-    def jenkinsSlaveImage = parameters.get('jenkinsSlaveImage', getDefaultSlaveImage(cloud))
-    def namespace = parameters.get('namespace', getDefaultSlaveNamespace(cloud))
+    def jenkinsSlaveImage = parameters.get('jenkinsSlaveImage', pipelineVars.centralCIjenkinsSlaveImage)
```

Comment:
This doesn't need to block the MR, but perhaps moving to v6 is a good time to rename all occurences of `jenkinsSlave` to `jenkinsAgent`?